masterlist.yaml

```
@@ -1672,6 +1672,17 @@ plugins:
     group: *fixesGroup
     msg: [ *alreadyInUFO4P ]
 
+  - name: 'DisabletheCaravan.esl'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/43961'
```

We add trailing slashes to URLs were possible for consistency & convenience (it makes it easier to search for, e.g., Nexus page IDs):

```suggestion
      - link: 'https://www.nexusmods.com/fallout4/mods/43961/'
```